spec/octokit/client/repositories_spec.rb

```diff
@@ -62,23 +62,6 @@
     end
   end
 
-  describe '.edit_repository', :vcr do
```

Comment:
Simplified and moved this test down into the `with repository` group - there was also a new cassette as a result of this change.